avm/src/vm/deploy.rs

```diff
@@ -201,7 +206,7 @@ pub async fn status() {
 
   let mut data: Vec<Vec<&dyn Display>> = vec![];
   for app in &mut apps {
-    data.push(vec![&app.id, &app.status, &app.url, &app.cloudProvider, &app.cloudAlias]);
+    data.push(vec![&app.id, &app.url, &app.cloudProvider, &app.cloudAlias]);
```

Comment:
Why does the `status` subcommand no longer have a status field?

Reply:
"info" is probably more accurate. Now that we wait for vm instances to spin up and down, the status is always "running". When we have SSH access or store the status of the cluster we will probably want to do something more elaborate to provide an actual "status". We could require the user to provide a `health` endpoint that we check through the main HTTPS url though, but if the user doesn't have that endpoint the app would show up as `unavailable`

Reply:
Discussed offline we decided to go with calling the command `info` for now